docker-compose.yml

```diff
@@ -138,6 +136,8 @@ services:
     networks:
       app_net:
         ipv4_address: 172.16.10.8
+    ports:
+      - 6969:6969 # for admin page, delete later
```

Comment:
@mariekart42 @kris-bo so you can access the admin page, need to be removed for production

Reply:
even with this, i still can't reach the admin page